### DISCUSSION TITLE
Is GS1 datamatrix format supported

### DISCUSSION THREAD

**Discussion by blagtout** [adrianschlatter/ppf.datamatrix#4] [Q&A] [answered]
dear @adrianschlatter , your module is super good.
Maybe I missed it, but I wanted to encode GS1 Datamatrix format, that includes normally FNC1 separators.
Is it possible to include them in the current version of the module?
here an example of a GS1 code : (01)99012345670019(3102)001225(15)190521(10)Ab1234
where (01) , (3102) ... are Application Identifiers
Thanks for your feedback
seb

**Comment by adrianschlatter** [comment, marked as answer]
Dear @blagtout

I'm glad you like ppf.datamatrix. Unfortunately, it currently does not support the FNC1 character. Also, it is unlikely that support will be added in the near future.

**Reply by blagtout** [reply]
thanks Adrian
